myEnigma/myEnigma.ino

```diff
@@ -1927,7 +1936,7 @@ void setup() {
   char strBuffer[]="PR X";
 
   Serial.begin(38400);
-  Serial.print(F("My enigma v"));
+  Serial.print(F("MeinEnigma v"));
```

Comment:
Initially the name was myEnigma but that is already used so later I changed it to meinEnigma, guess I missed this spot